src/page/Dashboard/components/StudyLogTable.tsx

```
@@ -0,0 +1,136 @@
+          <p className="typography-body-small-r mt-2">
+            타이머를 시작하여 첫 번째 기록을 만들어보세요!
+          </p>
+        </div>
+      ) : (
+        <section aria-label="학습 기록 목록">
+          <div className="overflow-x-auto">
+            <header
+              className={`bg-primary-10 grid ${GRID_COLS} items-center rounded-t`}
+              role="row"
+            >
+              <span className="typography-body-small-s text-secondary-indigo px-4 py-3">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<tr>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)